### ISSUE TITLE
Support RTP/UDP streams as another input

### ISSUE THREAD

**Issue by mincequi** [badaix/snapcast#909]
As I am writing and maintaining a bluetooth audio daemon (https://github.com/mincequi/cornrow) there had been a request to integrate my tool into snapcast. As I can see, there are integrations for stdout, tcp, named pipes. I am not quite convinced, that a TCP connection is the proper way to deliver audio and I would like to see RTP/UDP support with SAP/SDP session announcement.

I can implement this on my own and do a pull request. However, I would just start this effort, if you are also willing to integrate that?

Thanks in advance.

**Comment by gtsteffaniak** [comment]
This has been suggested many times, I agree it would alleviate many problems but at great development cost. I believe the official stance is that snapcast uses TCP and if you want a UDP option you will need to look elsewhere.

**Comment by mincequi** [comment]
Yes, UDP is the proper way to do realtime communication. But why should the development cost be so high? I could actually do that and i would stick to RTP/UDP for now as an another input signal besides stdout, named pipes, etc. It should not replace the official protocol that snapcast uses internally.

**Comment by gtsteffaniak** [comment]
Yes it would have been the correct way to do things I'd done from the beginning, but here we are, still using it anyways. 

It would be easier to write a new program with UDP than it would be to rewrite this one for UDP...
